src/workerd/jsg/resource.h

```diff
@@ -957,6 +957,13 @@ class ResourceWrapper {
                          v8Str(isolate, "FinalizationRegistry"_kj,
                                v8::NewStringType::kInternalized)));
 
+    // Capture a reference to the Error.captureStackTrace method for use with DOMException.
+    auto name = v8StrIntern(isolate, "captureStackTrace");
```

Comment:
I'm not familiar with Private api namespace, but should we add some namespacing here? like "workerd_captureStackTrace"?

Reply:
I don't think there's much of a need currently. The `ForAPI` mechanism here is specifically intended for this kind of use and gives us essentially an embedder-private namespace. If we were embedding other v8 consumers such as chromium or something then additional namespacing would be warranted. If this does become an issue later we can always add the additional namespacing without risking anything breaking.